beacon-chain/rpc/validator/server.go

```diff
@@ -89,7 +90,8 @@ func (vs *Server) WaitForActivation(req *ethpb.ValidatorActivationRequest, strea
 
 	for {
 		select {
-		case <-time.After(6 * time.Second):
+		// Pinging every slot for activation.
+		case <-time.After(time.Duration(params.BeaconConfig().SecondsPerSlot) * time.Second):
```

Comment:
Activation only occurs on a per-epoch basis, the rate we log out here is only a UX decision.